Documents/public-api.md

```diff
@@ -0,0 +1,36 @@
+# HMDA Platform Public API
+
+This documenatation describes de public HMDA Platform HTTP API
+
+## Institutions
+
+### Search
+
+* `/institutions?domain=<domain>`
+
+   * `GET` - Returns a list of institutions filtered by their email domain. If none are found, an HTTP 404 error code (not found) is returned
+
+   Example response, with HTTP code 200:
+
+   ```json
+   {
+     "institutions":
+     [
+        {
+          "id": "0",
+          "name": "Bank 0",
+          "domains": ["test@bank0.com"],
+          "externalIds":[
+            {
+              "value": "1234",
+              "name": "occ-charter-id"
+            },
+            {
+              "value": "1234",
+              "name": "ncua-charter-id"
+            }
+          ]
+        }
+     ]
```

Comment:
That wasn't actually part of this PR.  It came from #798...but I fixed it here (https://github.com/cfpb/hmda-platform/pull/808/commits/31426276a9c2239c69061fa37e9675393591786c) anyway, just for you @awolfe76! :)